File: week-02/day-03/takes_longer.py
# When saving this quote a disk error has occured. Please fix it.
# Add "always takes longer than" to the StringBuilder (quote) between the words "It" and "you"

quote = "Hofstadter's Law: It you expect, even when you take into account Hofstadter's Law."
you = quote.find("you")
quote = quote[:you] + "always takes longer than " + quote[you:]
print(quote)

File: week-07/day-03/takes_longer.js
'use strict';
// When saving this quote a disk error has occured. Please fix it.
// Add "always takes longer than" to between the words "It" and "you"

let quote = "Hofstadter's Law: It you expect, even when you take into account Hofstadter's Law."

let index = quote.indexOf("you");

quote = quote.slice(0, index) + "always takes longer than " + quote.slice(index);

console.log(quote);